src/runtime.rs

```diff
@@ -478,6 +486,9 @@ pub unsafe fn sigprocmask(how: How, set: Option<&Sigset>) -> io::Result<Sigset>
 
 /// `sigpending()`—Query the pending signals.
 ///
+/// If this is ever exposed publicly, we should think about whether it should
+/// mask out signals reserved by libc.
```

Comment:
Probably similar to how runtime signals are handled --- use libc sigrt in public API, but leave "raw" syscall which doesn't even care about libc in semi-public API

Reply:
Yes, I imagine it would look something like that.